### PR TITLE
[MOVE/#109] 세연 담당 뷰 패키징 재구성

### LINKED ISSUE #109: [MOVE] 세연 담당 뷰 / 패키징 재구성
## 🚩 TO DO
- [x] 패키징 수정
- [x] service, datasource, datasourceimpl, repository, repositoryimpl 합치기

### PR DESCRIPTION
**Pull request by crownjoe**
- closed #109

## *⛳️ Work Description*
- 패키징 수정

## *📢 To Reviewers*
- 수정 완료

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -96,13 +96,13 @@
...
-            android:name="com.going.presentation.entertrip.starttrip.invitetrip.CreateTripActivity"
+            android:name="com.going.presentation.entertrip.invitetrip.invitecode.CreateTripActivity"
             android:exported="false"
             android:screenOrientation="portrait" />
 
         <activity
-            android:name="com.going.presentation.entertrip.starttrip.StartTripSplashActivity"
-            android:exported="false"
+            android:name="com.going.presentation.entertrip.StartTripSplashActivity"
+            android:exported="true"
```
Comment (leeeyubin) [inline app/src/main/AndroidManifest.xml]: 에헴,,,,

File: data/src/main/java/com/going/data/service/EnterTripService.kt [hunk 2]
```diff
@@ -2,13 +2,22 @@ package com.going.data.service
...
     @POST("/api/trips/verify")
     suspend fun postEnterTrip(
         @Body request: EnterTripRequestDto,
     ): BaseResponse<EnterTripResponseDto>
+
+    @POST("/api/trips/{tripId}/entry")
+    suspend fun postStartInviteTrip(
+        @Path("tripId") tripId: Long,
+        @Body request: StartInviteTripRequestDto,
+    ): BaseResponse<StartInviteTripResponseDto>
```
Comment (leeeyubin) [inline data/src/main/java/com/going/data/service/EnterTripService.kt:22]: 오! 수고하셨어용

File: presentation/src/main/java/com/going/presentation/entertrip/invitetrip/finish/InviteFinishActivity.kt [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-import com.going.presentation.entertrip.starttrip.createtrip.EnterTripActivity.Companion.END
-import com.going.presentation.entertrip.starttrip.createtrip.EnterTripActivity.Companion.START
-import com.going.presentation.entertrip.starttrip.createtrip.EnterTripActivity.Companion.TITLE
-import com.going.presentation.entertrip.starttrip.createtrip.EnterTripActivity.Companion.TRIP_ID
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity.Companion.DAY
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity.Companion.END
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity.Companion.START
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity.Companion.TITLE
+import com.going.presentation.entertrip.createtrip.choosedate.EnterTripActivity.Companion.TRIP_ID
```
Comment (leeeyubin) [inline presentation/src/main/java/com/going/presentation/entertrip/invitetrip/finish/InviteFinishActivity.kt:12]: 폴더링 수고하셨습니둥

**Review by leeeyubin — APPROVED** [review]
좋아요!!!! LGTM~

**Review by Marchbreeze — APPROVED** [review]
굿 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

File: presentation/src/main/java/com/going/presentation/onboarding/splash/SplashActivity.kt [hunk 4]
```diff
@@ -5,6 +5,7 @@ import android.content.Intent
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
+import android.util.Log
```
Comment (chattymin) [inline presentation/src/main/java/com/going/presentation/onboarding/splash/SplashActivity.kt]: 지워줏셋욧

**Review by chattymin — APPROVED** [review]
굿!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Pull request merged**